### PR TITLE
[Merged by Bors] - better prints

### PR DESCRIPTION
**Pull request by almogdepaz**
## Motivation
better prints

### CODE REVIEW

File: miner/block_builder.go [hunk 1]
```diff
@@ -398,6 +398,11 @@ func (t *BlockBuilder) handleGossipAtx(data service.GossipMessage) {
 		commitmentStr = atx.Commitment.String()
 	}
 
+	h, err := atx.NIPSTChallenge.Hash()
+	if err != nil {
```
Comment (barakshani) [inline miner/block_builder.go]: what happens if you try to print `h.string` below but you got an error here?

Comment (almogdepaz) [reply miner/block_builder.go]: empty string

Comment (almogdepaz) [reply miner/block_builder.go]: i take that back, fixed.

**Review by barakshani — APPROVED** [review]

**Comment by almogdepaz** [comment]
bors merge

**Comment by almogdepaz** [comment]
bors merge

**Comment by almogdepaz** [comment]
bors merge

**Pull request closed without merging**